media/js/plugincheck/lib/plugincheck.js

```diff
@@ -14,17 +14,19 @@
      */
     function setJavaVersion() {
         var type;
-        var installedVersion;
+        var installedVersion = '0.0.0'; // fallback is better than returning undefined?
```

Comment:
Switching to `unknown` worked for me locally, but totally understand if it should be a version number for dependent functions.